### PR TITLE
Weno eRx fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
Fixed imports- had to redo drug paid import sql file as old was breaking transaction!
Redid pharmacies import and wrapped in transaction.
Fixed NDC (drug_id) not being saved with prescription or populating json.
Fixed qualifier and provider id syntax for populating transmission json.
Maybe others!

### CODE REVIEW

**Comment by sjpadgett** [comment]
@juggernautsei After all this and checking json sent to ensure all elements are populating I still don't get a response from weno server. It just sends (I guess it sends as all I see is whats in console) and that's it. If I just click transmit after sending a script then weno replys with invalid message as expected. What is weno's expected behavior???

**Comment by bradymiller** [comment]
@sjpadgett , Nice work. Just let me know when you want me to bring this in. -brady

**Comment by sjpadgett** [comment]
@bradymiller @juggernautsei Another update, the main going on's:
Redo of pharmacy import to match recent csv change (really just back to original).
Restyle imports admin and added import of city and state to narrow imported quantities some.
Fix for auto select of xmit checkboxs and disable not active scripts.
Change toggle action on txMark otherwise they are never cleared and always are selected.(code give a better idea)
The biggy here is behavior change of ajax sending to weno server. Trying to send update to UI can cause timing problems so I changed to batching ajax and UI writing using apply and promises. Now appears to always send and receive as expected.
I've tested fairly well but rec someone else from different setup....
Sherwin, I rec changing your prescription status updates to db until after weno gives success otherwise on fail the prescription state is as if the script was a success...
This should now work once weno messaging issue is figured out

File: interface/weno/admin.php [hunk 1]
```diff
@@ -50,26 +55,31 @@
...
-<form method="post" action="import_pharmacies.php" >
-    <div class="col-lg-2">
-    <?php echo generate_form_field(array('data_type'=>$GLOBALS['state_data_type'],'list_id'=>$GLOBALS['state_list'], 'field_id'=>'state'), ''); ?>
-    </div>
+    <form method="post" action="import_pharmacies.php" class="form-inline text-center">
 
+        <div class="row text-center">
+            <div class="form-group well col-lg-5 text-center">
+                <?php echo generate_form_field(array('data_type' => $GLOBALS['state_data_type'], 'list_id' => $GLOBALS['state_list'], 'field_id' => 'state'), ''); ?>
+                <input type="text" class="form-control" id="form_city" name="form_city" placeholder="Enter City"><br/><br/>
```
Comment (bradymiller) [inline interface/weno/admin.php]: xla() of 'Enter City'
(I finally found one :) )

**Comment by bradymiller** [comment]
Code looks good to me; just found 1 minor translation issue. If this is working for you, then we should bring this in (since otherwise WENO is not functional in the current codebase). Just give me the word, and I'll bring this into the codebase.

**Comment by sjpadgett** [comment]
Yep go ahead i'll finish working with Sherwin to get weno message solved.

**Comment by bradymiller** [comment]
Just brought into codebase. Thanks for fixing WENO! -brady

**Pull request merged**